docs/source/api/v2/asns.rst

```diff
@@ -172,3 +172,132 @@ Response Structure
 		"lastUpdated": "2019-12-02 21:49:08+00"
 	}}
 
+``PUT``
+=======
+Updates an existing :abbr:`ASN (Autonomous System Number)`.
+
+:Auth. Required: Yes
+:Roles Required: "admin" or "operations"
+:Response Type: Object
+
+Request Structure
+-----------------
+	+--------------+----------+---------------------------------------------------------------------------------+
```

Comment:
Missing `.. table::` directive

Reply:
These are parameters, it shouldn't even be a table. Fixed in fd7074989.